public/app/features/dashboard-scene/saving/provisioned/DashboardPreviewBanner.tsx

```diff
@@ -0,0 +1,37 @@
+import { Alert, Icon, Stack } from '@grafana/ui';
+import { DashboardPageRouteSearchParams } from 'app/features/dashboard/containers/types';
+
+interface DashboardPreviewBannerProps {
+  queryParams: DashboardPageRouteSearchParams;
+}
+export function DashboardPreviewBanner({ queryParams }: DashboardPreviewBannerProps) {
+  if (!queryParams.isPreview) {
+    return null;
+  }
+
+  const hasPrLink = Boolean(queryParams.prLink);
+  return (
+    <Alert
+      title={'Dashboard preview'}
+      severity={'success'}
```

Comment:
🤔  we may also want an info flavor of this -- so it is clear when viewing a branch too

Reply:
Hmm, yeah, this could actually be info as the main purpose is informational here. 
![Screenshot 2025-01-28 at 16 42 10](https://github.com/user-attachments/assets/1e5b8643-2972-4542-b75e-d633606c1825)